WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -145,7 +145,7 @@
                   <%= t '.team_leader' %>
                 </li>
                 <% current_user.teams_where_is_leader.each do |team| %>
-                  <li><%= link_to team.name, edit_team_path(team) %></li>
+                  <li><%= link_to team.name, edit_committee_team_path(team.committee.slug, team.slug) %></li>
```

Comment:
Let's make use of `to_param` method (that we have set up already) and just call `edit_committee_team_path(team.committee, team)`